src/__stories__/button-story.tsx

```diff
@@ -1,370 +1,113 @@
 import * as React from 'react';
-import {StorySection} from './helpers';
 import {
     Box,
-    Inline,
     ButtonDanger,
     ButtonLink,
     ButtonPrimary,
     ButtonSecondary,
     TextField,
     Stack,
-    skinVars,
-    Checkbox,
     Text2,
     IconPhotoCameraRegular,
     ResponsiveLayout,
 } from '..';
-import DoubleField from '../double-field';
-import {Title1} from '../title';
 
 export default {
     title: 'Components/Buttons',
+    parameters: {fullScreen: true},
+    argTypes: {
+        icon: {
+            options: ['none', 'left', 'right'],
+            control: {type: 'select'},
+        },
+    },
 };
 
-const BackgroundTheme: StoryComponent = ({children}) => {
-    const [isInverse, setIsInverse] = React.useState(false);
-    return (
-        <ResponsiveLayout isInverse={isInverse}>
-            <Box paddingY={24}>
-                <Box padding={8}>
-                    <Checkbox name="inverse" checked={isInverse} onChange={setIsInverse}>
-                        Inverse variant
-                    </Checkbox>
-                </Box>
-
-                {children}
-            </Box>
-        </ResponsiveLayout>
-    );
+const defaultArgs = {
+    text: 'Example',
+    loadingText: 'Loading text',
```

Comment:
I would keep the loadingText empty to have the button with the most common behaviour in the story.

Reply:
done